libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.h

```diff
@@ -116,4 +124,10 @@
  */
 - (void) updateLoggingPrefs;
 
+
+/**
+ * Clears unpublished events.
+ */
+- (void) clearAllEvents;
```

Comment:
@bcriscuolo sorry one more question, I can't find the method definition for this one?

Reply:
That is unnecessary, my apologies. I will remove this ASAP from this interface (SFS-iOS uses the `deleteAllEvents` method).